common/datavalues/Cargo.toml

```diff
@@ -20,7 +20,8 @@ common-macros = { path = "../macros" }
 primitive-types = "0.6.1"
 
 # Github dependencies
-opensrv-clickhouse = { git = "https://github.com/datafuselabs/opensrv", rev = "15786d3", package = "opensrv-clickhouse" }
+micromarshal = { git = "https://github.com/psiace/opensrv", rev = "2f3f08c", package = "micromarshal" }
```

Comment:
Please push a `v0.0.0` version of `opensrv` and `micromarshal` to crates.io for a placeholder.

Reply:
How about starting versioning `opensrv` so that we can build from `crates.io` instead of fetch from GitHub?

Reply:
Good idea, let's start from 0.1.0 :rocket: